src/store/index.js

```diff
@@ -1,14 +1,16 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
-import { getAuth, createUserWithEmailAndPassword, signInWithEmailAndPassword} from "firebase/auth";
+import { getAuth, createUserWithEmailAndPassword, signInWithEmailAndPassword } from "firebase/auth";
+import router from '../router';
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
     username: '',
     mailaddress: '',
-    password: ''
+    password: '',
+    wallet: '0',
```

Comment:
walletの値ですが、文字列ではなく数値で保持すると良いです。
```suggestion
    wallet: 0,
```